extensions/amp-video/0.1/amp-video.js

```diff
@@ -206,18 +312,25 @@ class AmpVideo extends AMP.BaseElement {
 
   /** @override */
   mutatedAttributesCallback(mutations) {
-    if (!this.video_) {
+    if (!this.isConnected_) {
       return;
     }
+
+    const {element} = this;
+    const resource = element.getResources().getResourceForElement(element);
+    const isLaidOut = !resource.isLayoutPending();
```

Comment:
wondering if `hasLoadedOnce()` is more accurate here. `isLayoutPending` would be true if `layoutCallback` has been called but failed therefore triggering the `fallback`.